app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/SignInScreen.kt

```diff
@@ -97,107 +100,114 @@ fun SignInScreen(
+                            horizontalArrangement = Arrangement.SpaceBetween,
+                        ) {
+                            Row(verticalAlignment = Alignment.CenterVertically) {
+                                Checkbox(
+                                    checked = checkedState,
+                                    onCheckedChange = { isChecked ->
+                                        signInViewModel.changeRememberPasswordState(isChecked)
+                                    },
+                                    colors = CheckboxDefaults.colors(checkedColor = AppTheme.colors.lightPink),
+                                )
+                                Text(
+                                    text = stringResource(id = R.string.remember_password),
+                                    fontSize = 12.sp,
+                                    fontFamily = FontFamily(Font(R.font.karm_light)),
+                                    fontWeight = FontWeight(700),
+                                    color = AppTheme.colors.darkPink,
+                                )
+                            }
                         }
-                    }
-                    Row(
-                        modifier = Modifier
-                            .height(32.dp)
-                            .fillMaxWidth(),
-                        Arrangement.End
-                    ) {
-                        TextButton(onClick = onForgotPasswordClick) {
-                            Text(
-                                text = stringResource(id = R.string.forgot_password),
-                                fontSize = 12.sp,
-                                fontFamily = FontFamily(Font(R.font.karm_light)),
-                                fontWeight = FontWeight(700),
-                                color = AppTheme.colors.darkPink,
-                            )
+                        Row(
+                            modifier = Modifier
+                                .height(32.dp)
+                                .fillMaxWidth(),
+                            Arrangement.End
+                        ) {
+                            TextButton(onClick = onForgotPasswordClick) {
+                                Text(
+                                    text = stringResource(id = R.string.forgot_password),
+                                    fontSize = 12.sp,
+                                    fontFamily = FontFamily(Font(R.font.karm_light)),
+                                    fontWeight = FontWeight(700),
+                                    color = AppTheme.colors.darkPink,
+                                )
+                            }
                         }
+                        AuthButton(
+                            modifier = Modifier
+                                .padding(horizontal = 32.dp, vertical = 16.dp)
+                                .fillMaxWidth(),
+                            shape = RoundedCornerShape(24.dp),
```

Comment:
You can use RoundedCorderShape defined on AppShapes.

Reply:
Fixed